dspace/config/registries/openaire4-relationships.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE relationships SYSTEM "relationships.dtd">
+
+<relationships>
+
+	<!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_creator.html 
+		The relation between a Publication and a Person regarding the authorship -->
+	<type>
+		<leftType>Publication</leftType>
+		<rightType>Person</rightType>
+		<leftwardType>isAuthorOfPublication</leftwardType>
+		<rightwardType>isPublicationOfAuthor</rightwardType>
+		<leftCardinality>
+			<min>0</min>
+		</leftCardinality>
+		<rightCardinality>
+			<min>0</min>
+		</rightCardinality>
+	</type>
+	<!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_creator.html 
+		The relation between a Publication and an Organization regarding the authorship -->
+	<type>
+		<leftType>Publication</leftType>
+		<rightType>Organization</rightType>
```

Comment:
Is an `Organization` considered different from an `OrgUnit`?  I know elsewhere we've used the term `OrgUnit` instead, e.g. in this sample `relationship-types.xml`: https://github.com/DSpace/DSpace/blob/master/dspace-api/src/test/data/dspaceFolder/config/entities/relationship-types.xml

If these are the same "relationship", then I'd prefer to rename this to `OrgUnit` so that we aren't creating `Organization` *and* `OrgUnit` entities in situations where a site wants to install both the default Entities and the OpenAIREv4 specific ones.

Reply:
Thank you Tim for point it out! We based this PR on that file, but for OpenAIRE purposes not all entities present on that file are required. There should be one relationship-types.xml in DSpace configurations directory by default.
Regarding the orgUnit vs Organization. An orgunit is a subset or part of an Organization. OpenAIRE, for instance, defines that a contributor can be a person or organization. So to be accurate with this alignment we proposed Organization. I discuss this with @josekarvalho and we agree to change it because you might also need to describe a publication or a work using a orgunit like in my case - FCCN is a unit of FCT organization.